beacon-chain/sync/batch_verifier.go

```diff
@@ -120,3 +171,51 @@ func performBatchAggregation(aggSet *bls.SignatureBatch) (*bls.SignatureBatch, e
 	}
 	return aggSet, nil
 }
+
+func (s *Service) validateWithKzgBatchVerifier(ctx context.Context, dataColumns []blocks.RODataColumn) (pubsub.ValidationResult, error) {
+	_, span := trace.StartSpan(ctx, "sync.validateWithKzgBatchVerifier")
+	defer span.End()
+
+	resChan := make(chan error)
+	verificationSet := &kzgVerifier{dataColumns: dataColumns, resChan: resChan}
+	s.kzgChan <- verificationSet
+
+	resErr := <-resChan
+	close(resChan)
```

Comment:
It would be safer if the sending side closed the channel, because receiving from a closed channel is fine, but sending to one causes a panic. This feedback will be more critical if you take the other piece of feedback to check `ctx.Done()` above, because in that instance you won't have read from `resChan`, so the other side could easily send to it after it is closed here and panic.